io.github.wazzaps.Fingerpaint.metainfo.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>io.github.wazzaps.Fingerpaint</id>
+
+    <name>Fingerpaint</name>
+    <summary>Draw using your laptop's touchpad</summary>
+
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>GPL-2.0</project_license>
+
+    <url type="homepage">https://github.com/Wazzaps/fingerpaint</url>
+    <url type="help">https://github.com/Wazzaps/fingerpaint/issues</url>
+    <url type="bugtracker">https://github.com/Wazzaps/fingerpaint/issues</url>
+
+    <description>
+        <p>
+            This utility lets you draw using your laptop's touchpad.
+            Pressing any key or clicking the touchpad will finish the drawing.
+            Your touchpad will not control the cursor while drawing, though external mice should still work.
+            It has support for both X11 (all desktop environments), and Wayland (just Gnome at the moment).
```

Comment:
so what happen if the user runs KDE Plasma on top of wayland?

Reply:
The tool fails to lock the cursor, though most functionality keeps working.